frontend/www/js/omegaup/components/arena/Course.test.ts

```diff
@@ -0,0 +1,61 @@
+jest.mock('../../../../third_party/js/diff_match_patch.js');
+
+import { mount } from '@vue/test-utils';
+
+import arena_Course from './Course.vue';
+
+describe('Course.vue', () => {
+  const currentAssignment = {
```

Comment:
nit:

```suggestion
  const currentAssignment: types.ArenaAssignment = {
```

cuando se declara una constante de interfaz, hay que intentar ponerle la anotación del tipo siempre.